### PR TITLE
Database alerts

### PR DESCRIPTION
**Pull request by mahirchavda**

### CODE REVIEW

File: cyences_app_for_splunk/default/data/ui/views/cs_mssql.xml [hunk 1]
```diff
@@ -62,6 +62,24 @@
         <latest>$tkn_time.latest$</latest>
       </search>
     </input>
+    <input type="multiselect" token="tkn_class_type" searchWhenChanged="true">
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/data/ui/views/cs_mssql.xml:65]: Is this filter relevant to other database dashboard? 
If so, please add filter in that dashboard as well.

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 2]
```diff
@@ -5827,11 +5827,85 @@ search = (`cs_mssql` (sourcetype="mssql:audit" OR sourcetype="mssql:audit:json")
...
-action.cyences_notable_event_action.system_compromised_drilldown = `cs_mssql` (sourcetype="mssql:audit" OR sourcetype="mssql:audit:json") class_type_desc="SQL LOGIN" host=$row.host$ | rename properties.* as *
+action.cyences_notable_event_action.system_compromised_drilldown = (`cs_mssql` (sourcetype="mssql:audit" OR sourcetype="mssql:audit:json")) OR (`cs_windows_idx` source="WinEventLog:Application" EventCode=33205 ) class_type_desc IN("SQL LOGIN","SQL USER") host=$row.host$ | rename properties.* as *
 action.cyences_notable_event_action.attacker_search = | stats count by user
-action.cyences_notable_event_action.attacker_drilldown = `cs_mssql` (sourcetype="mssql:audit" OR sourcetype="mssql:audit:json") class_type_desc="SQL LOGIN" user=$row.user$ | rename properties.* as *
+action.cyences_notable_event_action.attacker_drilldown = (`cs_mssql` (sourcetype="mssql:audit" OR sourcetype="mssql:audit:json")) OR (`cs_windows_idx` source="WinEventLog:Application" EventCode=33205 ) class_type_desc IN("SQL LOGIN","SQL USER") user=$row.user$ | rename properties.* as *
+action.cyences_send_email_action = 1
+action.cyences_notable_event_action.products = MSSQL
+
+[MSSQL - Role Changes]
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/savedsearches.conf:5838]: Let's add all 3 alerts for Oracle database as well. (it is okay if it is not 100% accurate, we can fix it when we find issue in different environment)

**Review by VatsalJagani — CHANGES_REQUESTED** [review]

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 3]
```diff
@@ -5827,11 +5827,85 @@ search = (`cs_mssql` (sourcetype="mssql:audit" OR sourcetype="mssql:audit:json")
...
+action.cyences_notable_event_action.contributing_events = (`cs_mssql` (sourcetype="mssql:audit" OR sourcetype="mssql:audit:json")) OR (`cs_windows_idx` source="WinEventLog:Application" EventCode=33205 ) class_type_desc IN("SERVER ROLE", "ROLE") | rename properties.* as *
+action.cyences_notable_event_action.system_compromised_search = | stats count by host
+action.cyences_notable_event_action.system_compromised_drilldown = (`cs_mssql` (sourcetype="mssql:audit" OR sourcetype="mssql:audit:json")) OR (`cs_windows_idx` source="WinEventLog:Application" EventCode=33205 ) class_type_desc IN("SERVER ROLE", "ROLE") host=$row.host$ | rename properties.* as *
+action.cyences_notable_event_action.attacker_search = | stats count by user
+action.cyences_notable_event_action.attacker_drilldown = (`cs_mssql` (sourcetype="mssql:audit" OR sourcetype="mssql:audit:json")) OR (`cs_windows_idx` source="WinEventLog:Application" EventCode=33205 ) class_type_desc IN("SERVER ROLE", "ROLE") user=$row.user$ | rename properties.* as *
+action.cyences_send_email_action = 1
+action.cyences_notable_event_action.products = MSSQL
```
Comment (hardikhdholariya) [inline cyences_app_for_splunk/default/savedsearches.conf:5873]: Just confirm the alert category with SOC and Compliance team and let me know. 
I'll add categories (SOC or Compliance or both) to all remaining alerts before the app release.

Comment (VatsalJagani) [reply cyences_app_for_splunk/default/savedsearches.conf:5873]: I think its Compliance, but Banking Institutes put it as part of Security in general. But we can discuss in the meeting and put it in that way.

**Review by VatsalJagani — APPROVED** [review]
